src/Stratis.Bitcoin/Controllers/Models/TransactionModel.cs

```diff
@@ -73,10 +73,16 @@ public TransactionVerboseModel(Transaction trx, Network network, ChainedHeader b
                 this.TxId = trx.GetHash().ToString();
                 this.Hash = trx.HasWitness ? trx.GetWitHash().ToString() : trx.GetHash().ToString();
                 this.Size = trx.GetSerializedSize();
-                this.VSize = trx.HasWitness ? trx.GetVirtualSize() : trx.GetSerializedSize();
+                this.VSize = trx.HasWitness ? trx.GetVirtualSize(network.Consensus.Options.WitnessScaleFactor) : trx.GetSerializedSize();
```

Comment:
I think it is inadvisable to ever use `GetSerializedSize` without any parameters, as it defaults to a particular protocol version internally to decide how the size is computed.

Reply:
I check NBitcoin and it uses `trx.GetSerializedSize()` without parameters everywhere. To fix this I would have to change it everywhere, maybe a different PR?

Reply:
Probably best to leave the other usages alone for the time being, then